Jenkinsfile

```diff
@@ -184,17 +184,16 @@ pipeline {
                 '''
               }
             }
-            stage("Test instantiation") {
+            stage('Test instantiation') {
               environment {
-                PATH = "/usr/local/Cellar/modules/4.1.2/bin:$PATH"
-                WORK_DIR = "/Volumes/data/working_dir_mac"
-                INPUT_DATA_DIR = "/Volumes/data_ci"
+                WORK_DIR = '/Volumes/data/working_dir_mac'
+                INPUT_DATA_DIR = '/Volumes/data_ci'
               }
               steps {
                 sh '''
                   source "${CONDA_HOME}/etc/profile.d/conda.sh"
                   conda activate "${WORKSPACE}/env"
-                  source /usr/local/opt/modules/init/bash
+                  source "${BREW_PREFIX}/opt/modules/init/bash"
```

Comment:
Do we need the quotes? (c.f l. 227)

Reply:
Not necessary. But I added lacking quotas for homogeneity (see 0d3aa91).